src/@primer/gatsby-theme-doctocat/components/storyBookEmbed.js

```diff
@@ -15,7 +15,7 @@ export function StorybookEmbed({framework = 'react', componentId, stories, heigh
     shortcuts: false,
     singleStory: Object.entries(stories).length <= 1,
     panel: false,
-    globals: `colorScheme:${colorScheme}`,
+    globals: `theme:${colorScheme}`,
```

Comment:
It seems that Storybook calls it `theme` instead of `colorScheme`:

![Screen Shot 2023-02-03 at 10 47 28](https://user-images.githubusercontent.com/378023/216491971-ae4dab1b-18ff-492f-8558-108b4a905d36.png)